src/AimInputComponent.cpp

```diff
@@ -18,27 +19,36 @@ AimInputComponent::~AimInputComponent() {
 void AimInputComponent::pollInput() {
     if (glfwGetKey(window, GLFW_KEY_I) == GLFW_PRESS) {
         toggleXRotation = true;
-        velocity = 1.0f;
+        rotationDirection = 1.0f;
     } else if (glfwGetKey(window, GLFW_KEY_K) == GLFW_PRESS) {
         toggleXRotation = true;
-        velocity = -1.0f;
+        rotationDirection = -1.0f;
     } else {
         toggleXRotation = false;
     }
 
     if (glfwGetKey(window, GLFW_KEY_J) == GLFW_PRESS) {
         toggleYRotation = true;
-        velocity = 1.0f;
+        rotationDirection = 1.0f;
     } else if (glfwGetKey(window, GLFW_KEY_L) == GLFW_PRESS) {
         toggleYRotation = true;
-        velocity = -1.0f;
+        rotationDirection = -1.0f;
     } else {
         toggleYRotation = false;
     }
 
+    toggleThrow = false;
     if (glfwGetKey(window, GLFW_KEY_SPACE) == GLFW_PRESS) {
-        toggleThrow = true;
+        if(!spacePressed){
```

Comment:
This is kind of awkward variable naming. It reads like If the `space` key is pressed then go into this condition and then in the inner condition you check to see that `spacePressed` is `false`. Maybe choosing a better variable name would be more readable? Or possibly set `pressStart = 0.0f` whenever it is inactive and do a check here to see if `pressStart` is some positive floating point.